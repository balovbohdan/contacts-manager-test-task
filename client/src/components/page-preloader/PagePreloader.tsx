import * as React from 'react';

import css from './PagePreloader.css';

export const PagePreloader = () =>
    <div className={css.main}>
        <div className={css.loader}/>
    </div>;