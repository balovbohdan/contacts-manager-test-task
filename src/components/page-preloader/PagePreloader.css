.main {
    width: 100%;
    height: calc(100vh - var(--header-height));

    display: flex;
    align-items: center;
    justify-content: center;

    background: var(--color-light-blue);
}

.loader {
    width: 50px;
    height: 50px;

    background-size: contain;
    background-repeat: no-repeat;
    background-position: center center;
    background-image: url(~@img/loader.svg);
}