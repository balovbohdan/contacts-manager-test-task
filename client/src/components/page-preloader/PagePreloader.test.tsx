import * as React from 'react';
import {shallow} from 'enzyme';

import {PagePreloader} from './PagePreloader';

describe('PagePreloader', () => {
    it('Should match snapshot', () => {
        const preloader = shallow(<PagePreloader/>);

        expect(preloader).toMatchSnapshot();
    });
});