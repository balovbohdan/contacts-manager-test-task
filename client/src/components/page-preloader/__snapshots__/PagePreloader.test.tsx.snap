// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagePreloader Should match snapshot 1`] = `
<div
  className="main"
>
  <div
    className="loader"
  />
</div>
`;
